src/i18n/fr/nav.ts

```diff
@@ -21,32 +25,33 @@ export default NavDictionary({
 	'core-concepts/astro-components': 'Composants',
 	'core-concepts/astro-pages': 'Pages',
 	'core-concepts/layouts': 'Layouts',
-	'guides/markdown-content': 'Markdown',
+	'guides/markdown-content': 'Contenu Markdown',
```

Comment:
In English this is “Markdown & MDX” so perhaps this should match:

```suggestion
	'guides/markdown-content': 'Markdown et MDX',
```